tests/test_cli.py

```diff
@@ -45,12 +46,17 @@ def image_with_metadata(image_with_exif_data):
     return image_with_exif_data
 
 
-def has_metadata(filepath, on_windows):
+def has_metadata(filepath, on_windows) -> bool:
     """Utility to check if a file has metadata."""
     with Image.open(filepath) as im:
         has_exif = dict(im.getexif()) != {}
         if on_windows:
             return has_exif
+        if RUNNING_ON_MACOS:
+            xattr_list = xattr(filepath).list()
+            return has_exif or (
+                xattr_list and set(xattr_list) != {'com.apple.provenance'}
+            )
```

Comment:
I think it's still relevant. Even if the image has just been created, MacOS still added this tag. But I tried to rewrite it into ignore all tag starts with `com.apple.`

Reply:
Were the tests failing for you in the main branch? If not, this is irrelevant because those fields aren't actually getting created when the pytest image file fixtures are created.

Reply:
Yes, they're failing for me in the main branch as well